src/teleop_twist_joy.cpp

```diff
@@ -80,7 +80,7 @@ TeleopTwistJoy::TeleopTwistJoy(rclcpp::node::Node::SharedPtr & node)
   rmw_qos_profile_t custom_qos_profile;
   custom_qos_profile.history = RMW_QOS_POLICY_HISTORY_KEEP_LAST;
   custom_qos_profile.depth = 50;
-  custom_qos_profile.reliability = RMW_QOS_POLICY_RELIABILITY_RELIABLE;
+  custom_qos_profile.reliability = RMW_QOS_POLICY_RELIABILITY_BEST_EFFORT;
```

Comment:
So, using the sensor default in combination with the new joystick node seems to make it work.  I'm not entirely sure why, but in any case I think we can switch to the default sensor profile.  I'll update this with that change soon.